custom_components/trakt_tv/configuration.py

```diff
@@ -12,6 +12,13 @@
 class Configuration:
     data: Dict[str, Any]
 
+    @property
+    def ignore_collected(self) -> bool:
+        try:
+            return self.conf["ignore_collected"]
+        except KeyError:
+            return False
+
```

Comment:
- Move together with the rest of anticipated functions
- `exclude` instead of `ignore` better matches other sources
- Rename to reflect that it's for anticipated only. Maybe `anticipated_exclude_collected`
- Pass identifier to the function to allow separate config for each
- Move the config under anticipated, e.g. `self.conf["anticipated"][identifier]["exclude_collected"]`